src/Middleware/RequestLimiter/sample/Controllers/HomeController.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.RequestLimiter;
+
+namespace RequestLimiterSample
+{
+    [RequestLimit(requestPerSecond: 10)]
```

Comment:
It's unlikely this would ever be hardcoded, I wonder how useful this is as an attribute. Maybe we need it for completeness.

Reply:
What do you mean hardcoded? I think it's probably useful to provide a default implementation that takes in common parameters such as "x request per second" since it's super easy to use. Otherwise, requiring the configuration of policies in ConfigureServices and specifying a policy name here may be overly complicated?

Reply:
I mean read from configuration and set based on the environment

Reply:
Ah yes, config driven limits. I haven't implemented that part yet.